catalog-info.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
```

Comment:
Just a general note, you should consider leaving comments in here with links to reference docs, either at the top or in front of specific sections.